classes/controller/class-enform-fields-list-table.php

```diff
@@ -100,7 +100,7 @@ public function get_columns() {
 	 *
 	 * @return string Content for column.
 	 */
-	protected function column_default( $item, $column_name ) {
+	protected function column_default( $item, $column_name ): string {
```

Comment:
it won't return anything if we don't give the proper `$item` parameter, so either add `return ''` at the end or as `default`, or `return null` and change the return type to `?string`
Same for `Enform_Fields_List_Table::column_default()`